clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2033,6 +2033,31 @@ def MemCpyOp : CIR_Op<"libc.memcpy"> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// FAbsOp
+//===----------------------------------------------------------------------===//
+
+def FAbsOp : CIR_Op<"libc.fabs", [Pure, SameOperandsAndResultType]> {
+  let arguments = (ins AnyFloat:$src);
+  let results = (outs AnyFloat:$result);
+  let summary = "Equivalent to libc's `fabs`";
+  let description = [{
+    Returns absolute value for floating-point input.
+
+    Examples:
+
+    ```mlir
+      %1 = cir.const(1.0 : f64) : f64
+      %2 = cir.libc.fabs %1 : f64
+    ```
+  }];
+
+  let assemblyFormat = [{
+    $src `:` type($src) attr-dict
+  }];
+  let hasVerifier = 0;
```

Comment:
This is probably not needed.

Reply:
We can remove it but all other operations seem to have it.

Reply:
Sounds good to leave it as is.